### ISSUE TITLE
Issue: Scoreboard not loading

### ISSUE THREAD

**Issue by PJNorrisS1** [moloch--/RootTheBox#558]
Hi, 

We have a strange one. If I create a new instance from 3.13.0 (latest), move our DB in to it, the scoreboard will not display. Just comes up with Loading... 
![image](https://user-images.githubusercontent.com/106666319/236513717-c9ade1a8-113a-41ad-a2a3-9f10fec222b2.png)
Nothing in log files or on console.

If I go to an existing instance with 3.13.0 - it's working fine. Even if I pull the latest and build the image, it will still load okay. I first thought it might be a requirements.txt dependency again.

Thoughts?

**Comment by g33kroid** [comment]
I am facing the same issue, and I noticed there is a JS that my Cloudflare blocks because of a Cross-Origin issue, but not sure how to solve it
```
# [ Server ]
origin = "ws://127.0.0.1:8888"
listen_port = 8888
listen_interface = "0.0.0.0"
session_age = 2880
x_headers = False
ssl = False
certfile = ""
keyfile = ""
admin_ips = []
autoreload_source = True
webhook_url = None
```
![image](https://user-images.githubusercontent.com/16447622/236704198-f8df2b65-4fe7-4249-b31e-b1844fd1c63f.png)

**Comment by eljeffeg** [comment]
`beacon.min.js` is not part of RootTheBox.  It's a analytic for cloudflare. https://developers.cloudflare.com/analytics/web-analytics

**Comment by eljeffeg** [comment]
@g33kroid in your case, I'd take a look at your origin (set it to your external hostname "wss://mydomain.com") and probably also turn `x_headers` to true.  Not sure if this is the same issue as @PJNorrisS1 but I will try to reproduce it.

**Comment by eljeffeg** [comment]
I haven't been able to reproduce this yet with a sqlite database.

**Comment by PJNorrisS1** [comment]
I’m sorry. It’s not you… it’s me. I use nginx proxy manager and when we set up the new host, we hadn’t enabled websocket support. Doh! I’m sorry.

**Comment by bdesforges** [comment]
@PJNorrisS1 can you please share how you were able to enable websocket support for your front end nginx server configuration ?

**Comment by PJNorrisS1** [comment]
Hey - I used NGinX Proxy Manager to host our front ends. On there is a checkbox for Websocket support. Sorry to be vague :-)

**Comment by eljeffeg** [comment]
On the RTB configureation, make sure your [x_headers](https://github.com/moloch--/RootTheBox/wiki/Configuration-File-Details#x_headers) are set to `True` and [origin](https://github.com/moloch--/RootTheBox/wiki/Configuration-File-Details#origin) is properly defined for your front end host.domain.

**Comment by bdesforges** [comment]
Thanks @eljeffeg . I did set your [x_headers](https://github.com/moloch--/RootTheBox/wiki/Configuration-File-Details#x_headers) are set to True (it was there by default) as well as played with a number of values for and [origin](https://github.com/moloch--/RootTheBox/wiki/Configuration-File-Details#origin) . 
I reviewed a number of issues in the repo, as well as the (incomplete?) wiki page you referenced in other posts as well. 

I think maybe what what I have trouble understanding is whether if these apply to a standalone RTB setup, or to a docker setup. I use the docker setup. 

When I access RTB with the local IP address, everything works great, but when I access it through my own reverse proxy, I get errors on the websocket requests. 
The value of origin doesn't seem to matter, as all the request for me, as the browser always end up requesting `wss://host.domain/...` which ends up failing. 

I'll see if I can make it work with the nginx manager to see if i can somehow extract the nginx configuration from it.
